File: lib/models/story.dart
class Story {
  final String id;
  final String title;
  final String fileName;
  final String quality;
  final String slug;
  final int duration;
  final bool isWatched;
  final bool isAmediateka;

  Story({
    required this.id,
    required this.fileName,
    required this.quality,
    required this.duration,
    required this.slug,
    required this.title,
    required this.isWatched,
    required this.isAmediateka,
  });

  Map<String, dynamic> toJson() {
    var map = <String, dynamic>{};
    map['id'] = id;
    map['title'] = title;
    map['quality'] = quality;
    map['duration'] = duration;
    map['slug'] = slug;
    map['fileName'] = fileName;
    map['is_watched'] = isWatched;
    map['is_amediateka'] = isAmediateka;
    return map;
  }
}

File: android/src/main/kotlin/uz/udevs/udevs_video_player/models/Story.kt
package uz.udevs.udevs_video_player.models

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class Story(
    @SerializedName("id")
    val id: String,
    @SerializedName("title")
    val title: String,
    @SerializedName("fileName")
    val fileName: String,
    @SerializedName("quality")
    val quality: String,
    @SerializedName("duration")
    val duration: Long,
    @SerializedName("slug")
    var slug: String,
    @SerializedName("is_watched")
    var isWatched: Boolean,
    @SerializedName("is_amediateka")
    var isAmediateka: Boolean,
) : Serializable